### PR TITLE
images: Use Fedora 34 as the base of fedora-testing

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-083939/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-083939/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-083939/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-084343/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-084343/

**Comment by mvollmer** [comment]
> Hmm, this has failed because the IMAGES variable in the bootstrap script had two words in it:

Different mirrors return different versions of the directory listing. Most of them have only one link to each image, but some have two (one behind the icon, one behind the name). Some even don't match our regexp at all.

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-57dvp.
Log: https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/image-refresh-2076-20210604-085555/

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -91,7 +91,7 @@ libreport-plugin-bugzilla \
 libreport-fedora
 "
 
-if [ "$IMAGE" = "fedora-34" ]; then
```
Comment (allisonkarlitskaya) [inline images/scripts/fedora.setup:94]: Can you completely invert the logic here?  ie: not only change the check to "is this fedora-33?", but also change the list above to contain `ntpsec` and change the code here to replace `ntpsec` with `ntp`, with appropriately updated comments?

That way when fedora-33 drops out, we can just delete this section.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: images/scripts/fedora.setup (hunk 1 above)
Comment (mvollmer) [reply images/scripts/fedora.setup:94]: Yes, very good point.

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-testing-20210604-091111

File: images/scripts/fedora.setup (hunk 1 above)
Comment (mvollmer) [reply images/scripts/fedora.setup:94]: Done.

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-01.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-094242/

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-testing-20210604-100000

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-094949/

**Comment by cockpituous** [comment]
image-refresh fedora-34 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-34-20210604-100707

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-100707/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-100707/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-2076-20210604-101616/

**Comment by cockpituous** [comment]
image-refresh fedora-33 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-33-20210604-103838

**Review by allisonkarlitskaya — APPROVED** [review]
Thanks!

Approved changes to the setup script, but you've got some failing tests :(

**Comment by martinpitt** [comment]
At this scale we need an image specific global allow_journal_message, looks too much for a naughty.

**Comment by mvollmer** [comment]
> At this scale we need an image specific global allow_journal_message, looks too much for a naughty.

Let's wait a bit for https://bugzilla.redhat.com/show_bug.cgi?id=1965743

**Comment by mvollmer** [comment]
> > At this scale we need an image specific global allow_journal_message, looks too much for a naughty.
> 
> Let's wait a bit for https://bugzilla.redhat.com/show_bug.cgi?id=1965743

Nah, let's ignore the message. See #https://github.com/cockpit-project/cockpit/pull/15896

**Pull request merged**